### PR TITLE
Run CircleCi without network traffic

### PR DESCRIPTION
**Pull request by wilzbach**
Sometimes CircleCi fails randomly as it tries to access the `dmd` binary before it's built. I hope that it's due to `-j5`.

For a failing run, see e.g. https://circleci.com/gh/dlang/dlang.org/121

Also `release` defines all these target, so it's better to use the more generic target.
As an additional protection against faulty network, I added `DIFFABLE=1` too.

### CODE REVIEW

**Comment by ZombineDev** [comment]
`-j` doesn't control the number of concurrent threads in a process, but the number of jobs (a.k.a processes). Regardless of whether DScanner has some sort of concurrency issue, changing the number of concurrent jobs won't affect it.

**Comment by wilzbach** [comment]
> -j doesn't control the number of concurrent threads in a process, but the number of jobs (a.k.a processes).

I know, but the dependencies in the dlang.org Makefile aren't 100% well-defined which sometimes leads to problems which I was hoping is the reason for CircleCi sometimes failing here.

>  Regardless of whether DScanner has some sort of concurrency issue, changing the number of concurrent jobs won't affect it.

Yes. DScanner isn't used nor run here ;-)

**Comment by ZombineDev** [comment]
Sorry, I was looking at too many PRs in a roll and I somehow thought that this PR is for phobos...

File: dlang.org.ddoc [hunk 1]
```diff
@@ -145,7 +145,7 @@ _=
 
 DBLOG_LATEST_AUTHOR=
 DBLOG_LATEST_DATE=
-DBLOG_LATEST_LINK=
+DBLOG_LATEST_LINK=https://dlang.org/blog
```
Comment (wilzbach) [inline dlang.org.ddoc]: This is required for builds with `DIFFABLE=0` and the `chmgen` URL checking:

```
Error while processing file: /home/circleci/dlang.org/web/index.html
object.Exception@chmgen.d(577): Empty URL
----------------
```

https://circleci.com/gh/dlang/dlang.org/487?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by wilzbach** [comment]
So this was failing due to `DIFFABLE=1` and `chmgen` checking for invalid links.
Funnily we don't run `chmgen` by default on DAutoTest.

Anyways, this is passing & ready now.

**Comment by andralex** [comment]
Hm, I think the makefile is good, or if not it should be made good. I suggest we take a look at it first, this looks like a step back.

**Comment by wilzbach** [comment]
> Hm, I think the makefile is good, or if not it should be made good. I suggest we take a look at it first, this looks like a step back.

Ok. I rebased the PR, s.t. it only contains the `DIFFABLE=1` change.

**Comment by wilzbach** [comment]
I have rememered the other reason why `-j1` is useful - it makes the log _a lot_ easier to read (I just ran into this problem) -> re-added.

**Comment by wilzbach** [comment]
> I have rememered the other reason why -j1 is useful - it makes the log a lot easier to read (I just ran into this problem) -> re-added.

The log looks like this atm:

```
c++ -c -o../generated/linux/release/64/cgxmm.o -Wno-deprecated -Wstrict-aliasing -Werror -fno-exceptions -fno-rtti -D__pascal= -DMARS=1 -DTARGET_LINUX=1 -DDM_TARGET_CPU_X86=1 -m64 -fPIC -std=gnu++98 -O2 -Idmd/root -Idmd/tk -Idmd/backend -I../generated/linux/release/64 -Idmd -DDMDV2=1 -MMD -MF ../generated/linux/release/64/cgxmm.deps dmd/backend/cgxmm.c
  (CC)  BACK_OBJS  dmd/backend/cgsched.c
c++ -c -o../generated/linux/release/64/cgsched.o -Wno-deprecated -Wstrict-aliasing -Werror -fno-exceptions -fno-rtti -D__pascal= -DMARS=1 -DTARGET_LINUX=1 -DDM_TARGET_CPU_X86=1 -m64 -fPIC -std=gnu++98 -O2 -Idmd/root -Idmd/tk -Idmd/backend -I../generated/linux/release/64 -Idmd -DDMDV2=1 -MMD -MF ../generated/linux/release/64/cgsched.deps dmd/backend/cgsched.c
.generated/ddoc -Df/home/circleci/dlang.org/web/dmd-freebsd.html macros.ddoc html.ddoc dlang.org.ddoc doc.ddoc .generated/2.078.1.ddoc  .generated/dblog_latest.ddoc .generated/dblog_latest2.ddoc dcompiler.dd freebsd.ddoc
Error: unrecognized switch '-'
posix.mak:507: recipe for target '/home/circleci/dlang.org/web/dmd-freebsd.html' failed
make: *** [/home/circleci/dlang.org/web/dmd-freebsd.html] Error 1
make: *** Waiting for unfinished jobs....
  (CC)  BACK_OBJS  dmd/backend/cod1.c
c++ -c -o../generated/linux/release/64/cod1.o -Wno-deprecated -Wstrict-aliasing -Werror -fno-exceptions -fno-rtti -D__pascal= -DMARS=1 -DTARGET_LINUX=1 -DDM_TARGET_CPU_X86=1 -m64 -fPIC -std=gnu++98 -O2 -Idmd/root -Idmd/tk -Idmd/backend -I../generated/linux/release/64 -Idmd -DDMDV2=1 -MMD -MF ../generated/linux/release/64/cod1.deps dmd/backend/cod1.c
  (CC)  BACK_OBJS  dmd/backend/cod2.c
...
```

https://circleci.com/gh/dlang/dlang.org/633

**Comment by andralex** [comment]
Please add a comment before the line to explain the rationale of `-j1`. Thanks!

**Pull request merged**